buzzmobile/sense/lidar/lidar.py

```diff
@@ -45,9 +76,26 @@ def genLidarImage(lidarPoints):
 
     #cv2.fillConvexPoly(matrix, np.array(polyPoints, np.int32), 0)
     cv2.fillPoly(matrix, [np.array(polyPoints, np.int32)], 0)
-    cv2.imshow("yo", matrix)
+    '''cv2.imshow("yo", matrix)
     cv2.waitKey(0)
-    cv2.destroyAllWindows()    
+    cv2.destroyAllWindows()'''
+    return matrix
+
+
+#genLidarImage([(1, 2.4), (2, 3.3), (1.5, 9), (3.5, 3.6)])
+
+def lidar_node():
+    #only you can prevent null pointers
```

Comment:
I appreciate the snarky but maybe also make it more explicit that the line below the comment is there to prevent null pointers